velox/functions/sparksql/Arithmetic.h

```diff
@@ -368,6 +368,68 @@ FOLLY_ALWAYS_INLINE static int8_t fromHex(char c) {
 }
 } // namespace detail
 
+template <typename T>
+struct WidthBucketFunction {
+  FOLLY_ALWAYS_INLINE bool call(
+      int64_t& result,
+      double value,
+      double min,
+      double max,
+      int64_t numBucket) {
+    // NULL would be returned if the input arguments don't follow conditions
+    // list belows:
+    // - `numBucket` must be greater than zero and be less than Long.MaxValue.
```

Comment:
Would you double check these conditions? They do not seem to match documentation.

Reply:
updated to align the doc